Xamarin.Forms.Core/Slider.cs

```diff
@@ -117,6 +135,28 @@ public double Value
 		}
 
 		public event EventHandler<ValueChangedEventArgs> ValueChanged;
+		public event EventHandler DragStarted;
+		public event EventHandler DragCompleted;
+
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public void SendDragStarted()
```

Comment:
should implement the interface explicitly

Reply:
@StephaneDelcroix Made the changes. Thanks!